web/src/games/checkers/game.ts

```diff
@@ -82,18 +39,18 @@ export const INITIAL_BOARD: Piece[] = [
   piece(20, 0),         null, piece(21, 0),         null, piece(22, 0),         null, piece(23, 0),         null,
 ];
 
-const MAN_DIRS: Coord[][] = [
-  [new Coord(-1, -1), new Coord(1, -1)],
-  [new Coord(-1, 1), new Coord(1, 1)],
+const MAN_DIRS: ICoord[][] = [
+  [coord(-1, -1), coord(1, -1)],
+  [coord(-1, 1), coord(1, 1)],
 ];
 
-const KING_DIRS: Coord[] = [new Coord(-1, 1), new Coord(1, 1), new Coord(-1, -1), new Coord(1, -1)];
+const KING_DIRS: ICoord[] = [coord(-1, 1), coord(1, 1), coord(-1, -1), coord(1, -1)];
 
 export function checkPosition(
   G: IG,
   playerID: string,
   piece: ICheckerPiece,
-  coord: Coord,
+  coord: ICoord,
```

Comment:
This name clashes with the "coord" function. Maybe rename the later to "createCoord"?

Reply:
Done.